{"name": "test_login_invalid_pwd", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'", "trace": "self = <tests.test_login.TestLogin object at 0x00000271F90A3550>\n\n    @pytest.fixture(autouse=True)\n    def classSetup(self):\n>       self.login_page = LoginPage(self.driver)\nE       AttributeError: 'TestLogin' object has no attribute 'driver'\n\ntests\\test_login.py:16: AttributeError"}, "description": "\n        User cannot log into Railway with invalid password.\n        :return:\n        ", "start": 1721027653197, "stop": 1721027653197, "uuid": "2be97ef1-da43-4bbd-a041-65873be5783a", "historyId": "100ccbdb43bda6e72274a719705810b8", "testCaseId": "100ccbdb43bda6e72274a719705810b8", "fullName": "tests.test_login.TestLogin#test_login_invalid_pwd", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DUYEN"}, {"name": "thread", "value": "25432-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}